databroker/core.py

```diff
@@ -1394,6 +1387,58 @@ def extract_dtype(descriptor, key):
         return reported
 
 
+class NoFiller(event_model.Filler):
```

Comment:
It feels like this class should be moved to `event-model`. Are there any reasons why not?

Reply:
https://github.com/bluesky/event-model/pull/127